src/components/main/defaultStates.ts

```diff
@@ -0,0 +1,35 @@
+export const defaultCurrentWeather = {
```

Comment:
need type definition here

Reply:
ok:)
but if I have default state, which don't mathes with type, which defined in generic (like this `useState<CurrentWeatherType>(defaultCurrentWeather)`) I will have a mistake and can't compilate project.

Reply:
that's why we are using typescript - to catch such errors :)